device/arista/x86_64-arista_7050_qx32/platform.json

```diff
@@ -1,7 +1,23 @@
 {
     "chassis": {
         "name": "DCS-7050QX-32",
-        "components": [],
+        "components": [
+	    {
+                "name": "Aboot()"
+            },
+            {
+                "name": "Scd(addr=0000:02:00.0)"
+            },
+            {
+                "name": "Ucd90120A(addr=4-004e)"
+            },
+            {
+                "name": "Ucd90120A(addr=8-004e)"
+            },
+            {
+                "name": "CrowSysCpld(addr=2-0023)"
+            }
+	],
```

Comment:
For this product, this is the current list of components:
```
Scd(addr=0000:04:00.0)
Ucd90120A(addr=6-004e)
Ucd90160(addr=10-004e)
```

Reply:
fixed